frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -317,6 +317,76 @@ function ReaderBookmark:updateHighlightsIfNeeded()
     self.ui.doc_settings:saveSetting("bookmarks_version", 20200615)
 end
 
+local function split(str, pat)
```

Comment:
We already have a split function in util: `util.gsplit()`, a more versatile one, so no need to add another one here.
It's not strictly usable as you use this one to make a table, but as then, you iterate that table, you can use it. It's used in frontend/apps/reader/modules/readertypography.lua and frontend/ui/bidi.lua and frontend/device/kobo/device.lua if you want to see how to use it.